changelog.md

```diff
@@ -1,3 +1,6 @@
+# 3.14.1
+- [FIXED] Add support for Babel/ES6 Modules [#4881](https://github.com/sequelize/sequelize/issues/4881)
```

Comment:
Change the wording to `Added support for Babel/ES6 imports`, people might confuse this with ES6 classes :)